src/gatling/simulations/scenarios/Solicitor_PRL_FL401_CaseCreate.scala

```diff
@@ -0,0 +1,1348 @@
+    ======================================================================================*/
+
+    .group("XUI_PRL_FL401_180_WithoutNoticeCheckYourAnswers") {
+      exec(http("XUI_PRL_FL401_180_005_WithoutNoticeCheckYourAnswers")
+        .post(BaseURL + "/data/cases/#{caseId}/events")
+        .headers(Headers.xuiHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-event.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "#{XSRFToken}")
+        .body(ElFileBody("bodies/prl/fl401/PRLFL401WithoutNoticeCheckYourAnswers.json"))
+        .check(substring("anyOtherDtailsForWithoutNoticeOrder"))
+        .check(jsonPath("$.state").is("AWAITING_SUBMISSION_TO_HMCTS")))
+
+      .exec(http("XUI_PRL_FL401_180_010_ViewCase")
+        .get(BaseURL + "/data/internal/cases/#{caseId}")
+        .headers(Headers.xuiHeader)
+        .header("x-xsrf-token", "#{XSRFToken}")
+        .check(jsonPath("$.events[?(@.event_id=='withoutNoticeOrderDetails')]"))
+        .check(jsonPath("$.state.id").is("AWAITING_SUBMISSION_TO_HMCTS")))
+
+      .exec(Common.userDetails)
+    }
+
+    .pause(MinThinkTime, MaxThinkTime)
+
+  val ApplicantDetails =
+
+    /*======================================================================================
+    * Click the Applicant Details link
+    ======================================================================================*/
+
+    group("XUI_PRL_FL401_190_CreateApplicantDetailsEvent") {
+      exec(http("XUI_PRL_FL401_190_005_ApplicantDetailsTrigger")
+        .get(BaseURL + "/cases/case-details/#{caseId}/trigger/applicantsDetails/applicantsDetails1")
+        .headers(Headers.navigationHeader)
+        .check(substring("Manage cases")))
+
+      .exec(Common.configurationui)
+      .exec(Common.configJson)
+      .exec(Common.TsAndCs)
+      .exec(Common.configUI)
+      .exec(Common.userDetails)
+      .exec(Common.isAuthenticated)
+      .exec(Common.monitoringTools)
+
+      .exec(http("XUI_PRL_FL401_190_010_ViewCase")
+        .get(BaseURL + "/data/internal/cases/#{caseId}")
+        .headers(Headers.xuiHeader)
+        .header("x-xsrf-token", "#{XSRFToken}")
+        .check(jsonPath("$.events[?(@.event_id=='withoutNoticeOrderDetails')]"))
+        .check(jsonPath("$.state.id").is("AWAITING_SUBMISSION_TO_HMCTS")))
+
+      .exec(http("XUI_PRL_FL401_190_015_CreateApplicantDetailsEvent")
+        .get(BaseURL + "/data/internal/cases/#{caseId}/event-triggers/applicantsDetails?ignore-warning=false")
+        .headers(Headers.xuiHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-event-trigger.v2+json;charset=UTF-8")
+        .check(jsonPath("$.event_token").saveAs("event_token"))
+        .check(jsonPath("$.id").is("applicantsDetails")))
+
+      .exec(Common.userDetails)
+      .exec(Common.userDetails)
+
+      .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain(BaseURL.replace("https://", "")).saveAs("XSRFToken")))
```

Comment:
And again, do we even need this though as a token is being used in a previous request??